bayesian_exercises/golf_model_comparisons/golf.jl

```diff
@@ -0,0 +1,386 @@
+### A Pluto.jl notebook ###
+# v0.11.14
+
+using Markdown
+using InteractiveUtils
+
+# ╔═╡ 732bdf94-f6c8-11ea-35ae-490c5536996d
+begin
+	using Turing
+	using HTTP
+	using CSV
+	using Plots
+	using StatsFuns: logistic
+	using StatsPlots
+	using StatsBase
+end
+
+# ╔═╡ ce2eb27c-f6c8-11ea-02c8-d1dcefce145d
+begin
+	resp = HTTP.get("https://raw.githubusercontent.com/stan-dev/example-models/master/knitr/golf/golf_data.txt");
+
+	data = CSV.read(IOBuffer(resp.body), normalizenames = true, header = 3)
+	x, n, y = (data[:,1], data[:,2], data[:,3]);
```

Comment:
These names are not very descriptive. Let's add a brief explanation of each column in the dataset.